### PR TITLE
Added autoloader

### PR DESCRIPTION
**Pull request by naxvog**
Needed to rename `plugin.php` as it had no `class-` prefix before.

### CODE REVIEW

**Comment by tillkruss** [comment]
Can you ping when these are ready, or mark them as drafts until they are ready?

**Comment by naxvog** [comment]
I put all currently existing classes into the new namespace so this commit is ready.

File: redis-cache.php [hunk 1]
```diff
@@ -26,12 +26,16 @@
 
 define( 'WP_REDIS_VERSION', $meta['Version'] );
 
+require_once WP_REDIS_PLUGIN_PATH . '/includes/class-autoloader.php';
```
Comment (tillkruss) [inline redis-cache.php:29]: Is `WP_REDIS_VERSION` still being defined?

Comment (naxvog) [reply redis-cache.php:29]: `WP_REDIS_VERSION` is now defined dynamically to the version definition in the plugin header comment.

File: redis-cache.php [hunk 2]
```diff
@@ -26,12 +26,16 @@
 
 define( 'WP_REDIS_VERSION', $meta['Version'] );
```
Comment (tillkruss) [inline redis-cache.php:27]: @naxvog: Ich war wohl total blind 🙈

**Pull request merged**